### DISCUSSION TITLE
Android foreground service - update nofication body/title/buttons

### DISCUSSION THREAD

**Discussion by xsorifc28** [capawesome-team/capacitor-plugins#193] [Q&A] [answered]
Is it possible to update the notification details (button, title, body) after starting a foreground service?
I am using [Android foreground service plugin](https://github.com/capawesome-team/capacitor-plugins/tree/main/packages/android-foreground-service)

**Comment by robingenz** [comment, marked as answer]
This is not yet possible. Feel free to create a feature request.
